mbq/atomiq/producers.py

```diff
@@ -73,6 +73,15 @@ class SNSProducer(BaseProducer):
     required_dependencies = ['boto3']
 
     def _create_task(self, topic_arn, payload):
+        if not topic_arn or not isinstance(topic_arn, str):
```

Comment:
I think you're supposed to use `basestring` now for this `isinstance` check?

Reply:
oo I didn't know about this. Good point.

Reply:
Only if you want to merge this before Monday